### ISSUE TITLE
[Enhancement]: Save other player configurations along with progress.

### ISSUE THREAD

**Issue by Dr-Blank** [advplyr/audiobookshelf#1173]
### Describe the feature/enhancement

I usually listen to 2 or more titles at a time, and the speed for every title differs. Coming from maBook, I miss this feature where it would save the player preferences for each book along with the progress statistic. This could be a server-side feature or just for client, that is up to your discretion. I think this would be more sensible if it were to be client-side, since every client has a different use case, one might listen to on phone with different speed/loudness than may their tv or something.

For now, only the listening speed is the player configuration that could be stored for every title, but in future if you add more configuration, for example, **loudness** or **equalizer**, that could also benefit from this design choice as those settings would also be unique to every book.

**Comment by advplyr** [comment]
My first thought is this might make sense to store with the progress. In that case it could be used by any client

**Comment by Dr-Blank** [comment]
> My first thought is this might make sense to store with the progress. In that case it could be used by any client

Yeah, storing with progress makes sense. Although it need not be updated as frequently as progress is synced, as it would most probably be changed less often.

**Comment by dathbe** [comment]
I’d only add that it would be nice to have a global default setting that could be set in setttings and probably also with the option of setting the default with a button on the speed setting popup page.

**Comment by some-guy-23** [comment]
I commented on #1785 as well, but this ticket actually exactly matches what I would like. Per-book playback speed settings. Would love to have this!

**Comment by Bvdmgg** [comment]
> I hate to be that guy who asks for updates on an issue, but this does seem to be one of the more popular requested features and it's been 18 months since it was opened or got an update from a dev.
> 
> Has this been added to the roadmap at all? Its really the only feature I'm still waiting on.

I would love this as well!

**Comment by Van-Tiem** [comment]
https://github.com/advplyr/audiobookshelf/issues/1173#issue-1454534281

This is a similar request that I have. I also use that app, mAbook, and wish that Audiobookshelf app had the same features, but mostly just for the speed setting to be personalized to each audiobook. 
I'm hoping that this additional comment will encourage the Creator/support team to add this in the next update. 
Pretty please!! 🥺

**Comment by stilgen16** [comment]
> > I hate to be that guy who asks for updates on an issue, but this does seem to be one of the more popular requested features and it's been 18 months since it was opened or got an update from a dev.
> > Has this been added to the roadmap at all? Its really the only feature I'm still waiting on.
> 
> I would love this as well!

Please please add this feature! It's such a small annoyance to have to change speeds constantly, but it's an annoyance nonetheless

**Comment by some-guy-23** [comment]
> Please please add this feature! It's such a small annoyance to have to change speeds constantly, but it's an annoyance nonetheless

Can give this client / app a try (iOS + Android). It has per-book speed (among a ton of other features!) https://github.com/pounat/absorb

Very active development on it too!
